src/platform-includes/performance/opentelemetry-install/node.mdx

```diff
@@ -0,0 +1,16 @@
+```bash {tabTitle:npm}
+npm install @sentry/node @sentry/opentelemetry-node
+```
+
+```bash {tabTitle:yarn}
+yarn add @sentry/node @sentry/opentelemetry-node
+```
+
+<Note>
+
+Note that @sentry/opentelemetry-node depends on the following peer dependencies:
+
+- @opentelemetry/api, version 1.0.0 or greater
+- @opentelemetry/sdk-trace-base, version 1.0.0 or greater, or a package that implements it, like @opentelemetry/sdk-node
+
+</Note>
```

Comment:
Should the note about dependencies come before the install step?

Reply:
personally I find it better placed below (so the flow: this is how to install sentry. But make sure that you also have these dependencies installed). But we can of course still adjust this later as well! :)